CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.5.2] - 2020-02-24
+
+### Changed
+- Bump authn-k8s client to v0.16.1 (cyberark/conjur-authn-k8s-client#70)
+
+### Fixed
+- Log into RH container registry before pushing (#1149)
+- Updated Conjur integration tests to set `POSTGRES_HOST_AUTH_METHOD=trust` to
```

Comment:
This change is not worthy of an inclusion in the changelog tbh - it's an internal detail that shouldn't be externally visible I think.

Reply:
it is a change that was made in a PR in this version - I agree it's minor, and not relevant to end-users, but this is a log of all the changes.

I'm not sure whether we want to flag items like this (by prefacing them with `[test]` or something?) to preclude them from release notes, but I can see why we'd include them in the (contributor-facing) change logs

Reply:
I don't believe that changelogs are intended to be a list of all changes but only the notable ones that may impact the users. Even the [keepachangelog defines](https://keepachangelog.com/en/1.1.0/) it as:
```markdown
### What is a changelog?

A changelog is a file which contains a curated, chronologically ordered
list of notable changes for each version of a project.
```

If I was a user of this repo, this change would not impact me unless I was a developer and as such is probably below the threshold of a curated + notable change.